packages/wonder-blocks-modal/package.json

```diff
@@ -23,7 +23,6 @@
     "@khanacademy/wonder-blocks-icon-button": "^3.4.14",
     "@khanacademy/wonder-blocks-layout": "^1.4.12",
     "@khanacademy/wonder-blocks-spacing": "^3.0.5",
-    "@khanacademy/wonder-blocks-toolbar": "^2.1.35",
```

Comment:
**praise:** Thanks for cleaning this up.  Is it up to consumers to embed a `Toolbar` in the modal if they need it?

Reply:
That's right, and TBH I think this might be a leftover from Modal V1.